contracts/connectors/loantoken/LoanTokenLogicStandard.sol

```diff
@@ -6,12 +6,12 @@
 pragma solidity 0.5.17;
 pragma experimental ABIEncoderV2;
 
-import "./LoanTokenSettingsLowerAdmin.sol";
-import "../LoanTokenLogicStorage.sol";
-import "../interfaces/ProtocolLike.sol";
-import "../interfaces/FeedsLike.sol";
-import "../../../modules/interfaces/ProtocolAffiliatesInterface.sol";
-import "../../../farm/ILiquidityMining.sol";
+import "./modules/shared/LoanTokenSettingsLowerAdmin.sol";
```

Comment:
`import "./modules/shared/LoanTokenSettingsLowerAdmin.sol";`
because this is a separate module, why is it imported here?

Reply:
Removed already.